### PR TITLE
Migrate lambda* modules and tests

### PR DESCRIPTION
**Pull request by alinabuzachis**
Depends-On: https://github.com/ansible/zuul-config/pull/443

Migrate lambda* modules and tests

### CODE REVIEW

File: tests/integration/targets/lambda/tasks/tagging.yml [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+      CamelCaseKey: CamelCaseValue
+      pascalCaseKey: pascalCaseValue
+      snake_case_key: snake_case_value
+      New Key with Spaces: Updated Value with spaces
+      NewCamelCaseKey: CamelCaseValue
+      newPascalCaseKey: pascalCaseValue
+      new_snake_case_key: snake_case_value
+  # Mandatory settings
+  module_defaults:
+    lambda:
```
Comment (gravesm) [inline tests/integration/targets/lambda/tasks/tagging.yml:30]: Similar to the route53 problem, it seems like this needs to be `amazon.aws.lambda`.

Comment (alinabuzachis) [reply tests/integration/targets/lambda/tasks/tagging.yml]: Done, thanks!

File: plugins/modules/lambda_alias.py [hunk 2]
```diff
@@ -0,0 +1,327 @@
...
+  gather_facts: false
+  vars:
+    state: present
+    project_folder: /path/to/deployment/package
+    deployment_package: lambda.zip
+    account: 123456789012
+    production_version: 5
+  tasks:
+  - name: AWS Lambda Function
+    lambda:
```
Comment (abikouo) [inline plugins/modules/lambda_alias.py:68]:
```suggestion
    amazon.aws.lambda:
```

File: plugins/modules/lambda_alias.py [hunk 3]
```diff
@@ -0,0 +1,327 @@
...
+      code_s3_key: "lambda/{{ deployment_package }}"
+      local_path: "{{ project_folder }}/{{ deployment_package }}"
+      runtime: python2.7
+      timeout: 5
+      handler: lambda.handler
+      memory_size: 128
+      role: "arn:aws:iam::{{ account }}:role/API2LambdaExecRole"
+
+  - name: Get information
+    lambda_info:
```
Comment (abikouo) [inline plugins/modules/lambda_alias.py:83]:
```suggestion
    amazon.aws.lambda_info:
```

**Review by abikouo — APPROVED** [review]

**Review by gravesm — APPROVED** [review]

**Comment by goneri** [comment]
regate

**Comment by goneri** [comment]
regate

**Comment by goneri** [comment]
regate

**Pull request merged**